xc/device.ml

```diff
@@ -1408,6 +1417,8 @@ module Vusb = struct
     if device_model = Profile.Qemu_trad then
       raise (Internal_error (Printf.sprintf "Failed to plug VUSB %s because domain %d uses device-model profile %s." id domid (Profile.string_of device_model)));
     debug "Vusb plugged: vusb device %s plugged" id;
+    (* Need to reset USB device before passthrough to vm according to CP-24616 *)
+    call_usb_reset [ hostbus ; hostport ];
```

Comment:
I think `call_usb_reset` should take two arguments: `hostbus` and `hostport` and turning this into an argument for `forkexced` should be done in `call_usb_reset`.